ofl/sourcecodepro/METADATA.json

```diff
@@ -68,7 +68,70 @@
       "weight": 600,
       "filename": "SourceCodePro-Semibold.ttf",
       "copyright": "Copyright 2010, 2012 Adobe Systems Incorporated (http://www.adobe.com/), with Reserved Font Name 'Source'. All Rights Reserved. Source is a trademark of Adobe Systems Incorporated in the United States and/or other countries."
-    }
+    },
+    {
+      "name": "Source Code Pro",
+      "postScriptName": "SourceCodePro-BlackItalicItalic",
```

Comment:
METADATA.json is definitely wrong here. The PS name in the font itself is `SourceCodePro-BlackIt`.